### PR TITLE
fix: disable edit btn if user is not allowed (stations, sensors, mino…

### PR DESCRIPTION
**Pull request by ktalbi**
…red sites)

https://www.pivotaltracker.com/story/show/143052635

### CODE REVIEW

**Comment by Matheo13** [comment]
What is the aim, disable edition for not allow users? Then you can do it directly in ns_form.
Individuals are editable?

File: Front/app/modules/objects/detail.view.js [hunk 1]
```diff
@@ -138,6 +138,13 @@ define([
       formConfig.afterDelete = function(response, model){
         Backbone.history.navigate('#' + _this.model.get('type'), {trigger: true});
       };
+      formConfig.afterShow = function(){
+          // if user is not allowed to edit form (user roles) disable edit btn
```
Comment (romfabbro) [inline Front/app/modules/objects/detail.view.js:142]: it is not the edit button to disable but the delete button

**Comment by romfabbro** [comment]
not good yet

**Comment by Matheo13** [comment]
https://github.com/NaturalSolutions/ecoReleve-Data/pull/599

**Pull request closed without merging**